build.ps1

```diff
@@ -60,6 +63,10 @@ else {
     } else {
         ExecSafe { & powershell $DotNetInstallFile -InstallDir $DotNetDirectory -Version $DotNetVersion -NoPath }
     }
+    # Octopus Modification

```

Comment:
FWIW I think tentacle installs dot net versions using scripts in:
https://github.com/OctopusDeploy/OctopusTentacle/tree/main/build/test-scripts/integration/common

That also contains the script that is called to run the tests. In that way TC gets configured to "just" call a script in that. Its someone like an interface and the git commits have the implementations that may vary.

I don't know if that is better or worse than modifying this build.ps1 but its a thing we do.

Reply:
Integration tests and builds each have their own code paths unfortunately.